src/main/config/firecloud-orchestration-compose.yaml.ctmpl

```diff
@@ -15,6 +15,13 @@ app:
     - FIRECLOUD_URL_ROOT={{.Data.env_firecloud_url_root}}
     - GOOGLE_SECRET_JSON={{.Data.env_google_secret_json}}
     - ADMIN_REFRESH_TOKEN={{.Data.env_admin_refresh_token}}
+    - REFRESH_TOKEN_SECRET_JSON={{.Data.env_refresh_token_secret_json}}
```

Comment:
in Vault's dev env, env_refresh_token_secret_json and env_google_secret_json are equivalent. If they are intended to be the same, let's just use one? If they are intended to be different, Vault dev needs updating.